modules/doc/content/newsletter/2023/2023_04.md

```diff
@@ -41,6 +41,10 @@ disable them.
 
 [XYMeshLineCutter.md] was added to enable general line cutting for a 2D mesh that only contains TRI3 and QUAD4 elements in the XY-plane. The new mesh generator trims off the fraction of the input mesh that is located on one side of a given straight line. The elements/nodes near the cutting line of the mesh are processed to ensure a smooth cross-section instead of a "zigzag" cross-section as generated by [PlaneDeletionGenerator.md].
 
+### Quadrilateral Mesh Generation in Simple Curvilinear Domains
+
+[TransfiniteMeshGenerator.md] is a novel mesh generator that allows a range of 2D meshing strategies in curvilinear domains. This mesh generator addresses the need for `QUAD` element meshes within simple curvilinear domains, which can be flexibly "stitched" to other regions to construct complex templated meshes. The core algorithm is known as transfinite interpolation or the Gordon-Hull algorithm. The only requirement of the mesh generator is that the domain is bounded by two-by-two opposing edges of types prescribed by the user. The mesh generator accepts different types of domain edges, `LINE,` `DISCRETE,` `PARSED,` and `CIRCARC.` The `CIRCARC` option constructs an arc of a circle given two endpoints and either a third middle point or the deviation from a perfect circle. 
```

Comment:
```suggestion
[TransfiniteMeshGenerator.md] is a novel mesh generator that allows a range of 2D meshing strategies in curvilinear domains. This mesh generator addresses the need for `QUAD` element meshes within simple curvilinear domains, which can be flexibly "stitched" to other regions to construct complex templated meshes. The core algorithm is known as transfinite interpolation or the Gordon-Hull algorithm. The only requirement of the mesh generator is that the domain is bounded by pairs of opposing edges with types prescribed by the user. The mesh generator accepts different types of domain edges, `LINE,` `DISCRETE,` `PARSED,` and `CIRCARC.` The `CIRCARC` option constructs an arc of a circle given two endpoints and either a third middle point or the deviation from a perfect circle. 
```

This adjustment made things read a bit clearer for me. Let me know if you think this is less accurate than what you intended.